src/Equinor.Procosys.Library.WebApi/Controllers/Register/RegisterController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Equinor.Procosys.Library.Domain;
+using Equinor.Procosys.Library.Query.GetAllRegisters;
+using Equinor.Procosys.Library.WebApi.Misc;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using ServiceResult.ApiExtensions;
+
+namespace Equinor.Procosys.Library.WebApi.Controllers.Register
+{
+    [ApiController]
+    [Route("Registers")]
+    public class RegisterController : ControllerBase
```

Comment:
Should this be called RegistersController since the other controllers are plural?